operations/admin_all_configs.go

```diff
@@ -88,5 +97,23 @@ func fetchAndWriteConfig(c *legacyClient, project string) error {
 		return errors.Wrapf(err, "failed to write configuration for project %s", project)
 	}
 
+	if validate {
+		validationErrs, err := ac.ValidateLocalConfig(data)
+		if err != nil {
+			return errors.Wrap(err, "error validating config")
+		}
+
+		if len(validationErrs) > 0 {
+			grip.Infof("validation errors for %s:\n", project)
+			for _, err := range validationErrs {
+				level := "Error"
+				if err.Level == validator.Warning {
+					level = "Warning"
+				}
+				grip.Infof("[%s] %s", level, err.Message)
```

Comment:
can you implement message.Composer for the validation errors type (probably in the validator package? to avoid this loop?